### PR TITLE
PODS-8821 - Add Event 24/25 Manual Journey

### PR DESCRIPTION
**Pull request by LauraToddCodes**
[Intermediary PR](https://github.com/hmrc/pension-scheme-event-reporting-frontend/pull/291) renaming Event 25 to Event 24

### CODE REVIEW

**Review by PavelVjalicin — CHANGES_REQUESTED** [review]

File: conf/app.routes
Comment (PavelVjalicin) [inline conf/app.routes:1]: I think the following is missing from here:
GET         /report/remove-24                                                controllers.common.RemoveEventController.onPageLoad(waypoints: Waypoints ?= EmptyWaypoints, eventType: EventType = Event24)
POST        /report/remove-24                                                controllers.common.RemoveEventController.onSubmit(waypoints: Waypoints ?= EmptyWaypoints, eventType: EventType = Event24)

App crashes for me when trying to start a new event report for year 2022 and earlier.

**Comment by LauraToddCodes** [comment]
@PavelVjalicin Thanks for the review - I've pushed changes to resolve the issues with apostrophes in the messages, test routes in the prod.routes file and the missing `report/remove-24` route. Do you know if any changes are needed for filtering of events in the `EventSelectionController`?

File: app/controllers/EventSelectionController.scala [hunk 1]
```diff
@@ -85,24 +79,18 @@ class EventSelectionController @Inject()(val controllerComponents: MessagesContr
...
+  private def getFilteredOptions(ua: UserAnswers)(implicit messages: Messages): Seq[RadioItem] = {
     val eventsToRemove: Seq[EventSelection] = TaxYear.getSelectedTaxYear(ua).startYear >= "2024" match {
       case true => Seq(Event2, Event6, Event7, Event8, Event8A)
-      case false => Seq() // TODO: Include new events once developed Seq(Event25)
-    }
-    if (isEnabled) {
-      EventSelection.options(EventSelection.values.diff(eventsToRemove))
-    } else {
-      EventSelection.options(EventSelection.values)
+      case false => Seq(Event24)
```
[the remark this answers is not in the document]
Comment (jamdade-sharad) [reply app/controllers/EventSelectionController.scala:92]: @PavelVjalicin : 
This is correct, if selected tax year is 2024 onwards then need to hide Event2, 6, 7, 8 and 8A 
and before 2024 hide the Event24.

it means Event24(previously known as Event25) only visible year2024 onwards.

[the remark this answers is not in the document]
Comment (PavelVjalicin) [reply app/controllers/EventSelectionController.scala:92]: I've tested the above seems to work properly.

**Comment by PavelVjalicin** [comment]
> @PavelVjalicin Thanks for the review - I've pushed changes to resolve the issues with apostrophes in the messages, test routes in the prod.routes file and the missing `report/remove-24` route. Do you know if any changes are needed for filtering of events in the `EventSelectionController`?

I've tested this. Seems to work as intended.

**Review by PavelVjalicin — APPROVED** [review]
LGTM

**Pull request merged**